### DISCUSSION TITLE
심화 프로젝트 최종 회고

### DISCUSSION THREAD

**Discussion by yeji0512** [codeit-bootcamp-frontend/2-Weekly-Mission#447] [회고]
길었던 심화프로젝트의 끝이네요! 
너무 너무 고생 많으셨습니다 😊
기초, 중급, 심화 모두 무사히 마무리하셨으니 앞으로도 걱정 없겠어요 💪

마지막 프로젝트인 만큼 아쉬움이 남을 수도, 혹은 후련할 수도 있을 것 같은데요 
이번 최종 회고에서는 이를 편안하게 나누어볼 수 있는 자리가 되었으면해요! 

너무 고생 많으셨습니다 🥰

# 왜 회고를 해야하나요?
- **원활한 팀 프로젝트 진행을 위해, 회고가 좋은 도구가 되어줄거예요.** 
매주 팀 단위로 회고를 하며 어떤 점이 좋았고, 어떤 점이 아쉬웠고, 아쉬웠던 부분은 어떻게 개선해볼지 함께 이야기해보면 
그 다음주에는 더 나은 프로젝트를 진행할 수 있을거예요. 회고를 통해 함께 성장해보아요 🔥 
- **회고 시간만큼은 솔직한 생각, 마음을 터놓아보기! 😎** 
솔직한 회고를 통해 다음주는 더 즐겁게 프로젝트를 진행할 수 있을거예요.  

# 회고 방법
![image](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/144666341/b1f1cc83-7d24-4156-910d-3fc9b6bb7555)
회고는 **“KPT 회고”** 방식으로 진행해볼거예요. 아래 방법대로 회고해주시면 됩니다. 
- 팀원들과 모여 그 주의 팀 프로젝트 활동을 돌아보며, '좋았던 점(Keep) / 아쉬웠던 점(Problem) / 시도해볼 것(Try)'으로 항목을 나누어 회고해주세요.
    - Keep: 현재 우리팀이 잘 하고 있어서 계속 유지하면 좋을 것 같은 부분
    - Problem: 현재 우리팀이 가지고 있는 문제
    - Try: Problem에 적은 것을 해결하기 위해 우리 팀이 다음에 시도해볼만한 것 (혹은 Keep 중에서도 더 개선하거나 시도해보고 싶은 것)
- [KPT 회고 피그마 템플릿](https://www.figma.com/file/0NHIhlM9vhYZDrtZa7nJn8/%ED%9A%8C%EA%B3%A0-%ED%85%9C%ED%94%8C%EB%A6%BF?type=whiteboard&node-id=0%3A1&t=p6vXDVEjQntHKaNP-1)을 활용해서, 팀 회고를 진행하셔도 됩니다. 단 템플릿 사용시, 팀별로 복제해서 사용해주세요. 꼭 템플릿을 사용하지 않아도 되니, 팀별로 편한 방식으로 회고를 진행해주세요. 
- **이번주 일요일까지, 팀 회고한 내용을 정리하여 댓글로 올려주세요.** 
(댓글에 예시 회고를 올려드릴테니, 참고해주세요. 피그마 템플릿으로 진행하셨다면 캡처본을 올려주셔도 돼요.)

# 참고 글
- [KPT 회고란 무엇인가?](https://brunch.co.kr/@jinha0802/35)
- [팀문화의 탄생](https://techblog.woowahan.com/2677/)

**Comment by muyahho** [comment]
### ### 4팀

### ### Keep

리팩토링을 진행했습니다.
매일 회의록을 작성합니다.
소통이 원활히 진행됩니다.
문제 해결을 위한 협력이 활발합니다.
변함없이 열심히 하려고 합니다.
요구사항을 다 구현하고 빨리 잘 마무리했습니다. 

끝까지 본인이 맡은 부분을 책임지고 마무리 했다.

### ### Problem

컨벤션들을 제대로 지키지 않은 부분이 있는것 같습니다.
코드리뷰를 시간부족으로 꼼꼼히 못했습니다. 

### ### Try

컨벤션 제대로 지키기
적극적으로 코드 리뷰 하기
요구사항을 정확히 더 확인해서 리팩토링 하기
PR을 자주 올리기

끝!

**Comment by 9ummy9** [comment]
# 6팀


![image](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/148167964/9c820eef-43e2-4b26-9a64-4985a17549aa)



----


![tumblr_nqd7jxXR8O1rzk6edo1_500](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/148167964/648fa250-8da8-4850-9270-e6e907679592)


추후 남은 개발도 파이팅!
